src/styles/user-signup.css

```
@@ -0,0 +1,66 @@
+.SignUpForm {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+  }
+
+  .SignUpForm div {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+  }
+
+  .SignUpForm p {
+    font-size: 12px;
+    text-align: center;
+  }
+
+  .SignUpForm ::placeholder {
+    color: #989898;
+  }
+  
+  .SignUpForm form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .SignUpForm label {
+    margin: 10px 0;
+    padding-top: 20px;
+  }
+
+  .SignUpForm img {
+    width: 200px;
+    padding-top: 20px;
+    margin: auto;
+  }
+  
+  .SignUpForm input[type="text"],
+  .SignUpForm input[type="password"],
+  .SignUpForm input[type="email"],
+  .SignUpForm input[type="submit"] {
```

Youre creating rules for the input[type="submit"] selector then overwriting them straight away afterwards, may as well just remove that selector here and let the one underneath it do the work
```suggestion
  .SignUpForm input[type="email"] {
```